docs/Documentation/Applications/vasp.md

```diff
@@ -347,17 +328,30 @@ To run VASP, the following 4 input files are needed: POSCAR, POTCAR, INCAR, KPOI
 
     module purge
 
-    #Load OpenMPI VASP build and necessary modules
-    ml vasp 
+    #Load Intel MPI VASP build and necessary modules
+    ml vaspintel 
     ml slurm/21-08-1-1-o2xw5ti 
-    ml openmpi/4.1.1-6vr2flz
+    ml gcc/9.4.0-v7mri5d 
+    ml intel-oneapi-compilers/2021.3.0-piz2usr 
+    ml intel-oneapi-mpi/2021.3.0-hcp2lkf 
+    ml intel-oneapi-mkl/2021.3.0-giz47h4
 
     srun -n 32 vasp_std &> out
     ```
 
-### Vermilion
+??? note "Performance Notes"
+    The Intel MPI builds are reccomended over the Open MPI builds as they exhibit fastest performance.
+
+    Use at most 64 cores/node. On Swift, each node has 64 physical cores, and each core is subdivided into two virtual cores in a process that is identical to hyperthreading. Because of this, up to 128 cores can be requested from a single Swift node, but each core will only represent half of a physical core. 
+
+    On Swift, VASP is most efficiently run on partially full nodes. 
+
+    Unlike on Eagle, multiple jobs can run on the same nodes on Swift. If you are only using a fraction of a node, other users' jobs could be assigned to the rest of the node, which might deteriorate the performance. Setting "#SBATCH --exclusive" in your run script prevents other users from using the same node as you, but you will be charged the full 5AUs/node, regardless of the number of CPUs/node you are using. 
+
+## VASP on Vermilion
 
-??? example "Vermilion: VASP 6 (Intel MPI) on CPUs"
+#### CPU
+??? example "Sample job script: Vermillion - VASP 6 (Intel MPI)"
```

Comment:
I was not able to run this script.  As written, I get an error loading the vasp module:
```bash
[jmcfarla@vs-login-1 vasp-docs]$ module purge
[jmcfarla@vs-login-1 vasp-docs]$ ml vasp/6.3.1
Lmod has detected the following error: The following module(s) are unknown: "vasp/6.3.1"
```

I tried to work around this by using:
```bash
source /nopt/nrel/apps/210929a/myenv.2110041605
ml vaspintel
```
And reducing the number of tasks from 60 to 59.

However, even with those changes I get a slew or slurm errors, starting with:
```
slurmstepd: error: mpi/pmi2: invalid client request
slurmstepd: error: mpi/pmi2: request not begin with 'cmd='
slurmstepd: error: mpi/pmi2: full request is: 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```

I don't want to delay getting the rest of the updates from this PR merged, but I think at a minimum we should open a ticket about getting a working example script.

Reply:
I've created an issue to review all the vermillion modules.